wrangler-core/src/test/java/io/cdap/directives/parser/ParseAvroFileTest.java

```diff
@@ -52,6 +52,40 @@ public void testParseAsAvroFile() throws Exception {
     Assert.assertEquals(1495194308245L, results.get(1688).getValue("timestamp"));
   }
 
+  @Test
+  public void testAvroWithJsonFieldParsingWithParseJsonIgnoreError() throws Exception {
```

Comment:
these new tests are testing two directives. More specifically, these are not testing any change in behavior in the avro directive, they are just testing the changes in json directive.

Unit tests should be testing isolated components when possible. It would be better to enhance the parse json test. For example, if we remove the parse avro file directive, we could easily remove test coverage of the parse json directive accidentally.

Reply:
This was a problem customer had. We have other tests for catching parse-as-json problems. This was when a json in avro file have. Using redacted user data as test.